src/manifest.c

```diff
@@ -582,6 +582,16 @@ void subtract_manifests(struct manifest *m1, struct manifest *m2)
 			list1 = g_list_next(list1);
 			list2 = g_list_next(list2);
 
+			/* When both files are deleted, skip subtraction, since
```

Comment:
Can this be re-worded to mention that both files being deleted && having a same last_change value does not qualify them for subtractions still? Also a test for this would be nice :)